src/labels/CustomLabels.labels

```diff
@@ -1390,6 +1390,14 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <shortDescription>Installment Period cannot be converted to enhanced Recurring Donation format</shortDescription>
         <value>Recurring Donation Installment Period "{0}" cannot be converted to the Enhanced Recurring Donations format. Make sure the Installment Period is defined in NPSP Settings -> Recurring Donations -> Recurring Donation Custom Installment Periods.</value>
     </labels>
+    <labels>
+        <fullName>RD2_DataMigrationInvalidOpenEndedStatus</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Open Ended Status cannot be converted to enhanced Recurring Donation format</shortDescription>
+        <value>Recurring Donation Open Ended Status "{0}" cannot be converted to the Enhanced Recurring Donations format.</value>
```

Comment:
@sam-knox , can you please suggest text to display when RD legacy Open Ended Status cannot be converted into enhanced RD Status during data migration?

Reply:
@sam-knox ^

Reply:
Thanks @sam-knox , the change has been applied.